sidecar/proxy/envoy/manager.go

```diff
@@ -96,6 +96,15 @@ func NewManager(identity identity.Provider, conf *config.Config) (Manager, error
 		}),
 	}
 
+	if conf.ProxyConfig.TLS {
+		m.tlsConfig = &SSLContext{
+			CertChainFile:   conf.ProxyConfig.CertChainFile,
+			PrivateKeyFile:  conf.ProxyConfig.PrivateKeyFile,
+			CACertFile:      &conf.ProxyConfig.CACertFile,
+			GrpcHttp1Bridge: conf.ProxyConfig.GrpcHttp1Bridge,
+		}
```

Comment:
Why is the gRPC bridge under tls config ? And what happened to original tls config stuff ? Was it removed ? @GregHanson ?